src/mediaplayer.js

```diff
@@ -191,6 +191,7 @@ const isResponsiveSize = sizeProps => {
  * @param {Object} config
  * @param {String} config.type - The type of media to play
  * @param {String|Array} config.url - The URL to the media
+ * @param {String} [config.mimeType] - The MIME type of the media
```

Comment:
Do we really need a separate property? If you have a case in mind where `type` and `mimeType` are different, maybe add an example of their values in jsdoc here? 

[MediaInteraction uses `type` and doesn't set `mimeType` anyway.]

Reply:
The `type` is expected to be either `audio` or `video`. Nothing oblige to make it a MIME type, and this is a shorthand to also have the MIME type inside it. Usually, the MIME type is supplied together with the media source, in the `source` parameter. But the source can also be supplied through the `url` parameter. Since nothing ensure the type is having the MIME as well, we need to have a way to supplied it properly.

But yes, I agree, the documentation is minimalistic. I updated it.